iop/common/fhi.h

```diff
@@ -6,9 +6,10 @@
 
 #define FHI_FID_CDVD    0
 #define FHI_FID_ATA0    1
-#define FHI_FID_ATA1    2
-#define FHI_FID_MC0     3
-#define FHI_FID_MC1     4
+#define FHI_FID_ATAID0  2
+#define FHI_FID_ATA1    3
+#define FHI_FID_MC0     4
+#define FHI_FID_MC1     5
```

Comment:
When extending the number of file id's, make sure you also extend the `#define BDM_MAX_FILES 5` to `6` in fhi_bdm.h.

Reply:
Doh, its in another header file :D done